modules/tinymce/src/themes/silver/main/ts/ui/dialog/UrlInput.ts

```diff
@@ -146,7 +146,7 @@ export const renderUrlInput = (
     model: {
       getDisplayText: (itemData) => itemData.value,
       selectsOver: false,
-      populateFromBrowser: false
```

Comment:
Is it really this fragile I thought the specs where validated though boulder? But maybe that just ignores things it doesn't recognize if so should we harden boulder so that unknown things produces a validation error?

Reply:
Unfortunately yes... boulder just completely ignores any key it doesn't know about. As for changing it, that might be a good idea but it also has a lot of potential for causing a heap of failures as I suspect at the moment full specs are passed that would have a lot of unregistered properties.

Reply:
Boulder has a process for this already, but it isn't used by that many things (https://github.com/tinymce/tinymce/blob/310bafec6dbd453be7746cd2ecf5f5c40b18ed1b/modules/boulder/src/main/ts/ephox/boulder/api/StructureSchema.ts#L73) due to a potential performance hit.

Reply:
The top-level schemas use it (https://github.com/tinymce/tinymce/blob/310bafec6dbd453be7746cd2ecf5f5c40b18ed1b/modules/alloy/src/main/ts/ephox/alloy/spec/SpecSchema.ts#L34), but it doesn't inherit that setting all the way down, so you have to specify it every time you are composing a new object of fields.

Reply:
Worth noting this would likely be much less of a problem/less fragile if we had better boulder types... atm there's a lot if `any`'s in there.